common/test/acceptance/pages/lms/video/video.py

```diff
@@ -436,6 +437,7 @@ def speed(self, speed):
 
         """
         # mouse over to video speed button
+        self.wait_for_video_player_render()
```

Comment:
This synchronization really should not be in the speed button setter method. Instead it should be at the end of whatever the prior action's method was.

I'm looking through the codebase and it looks like the speed setter is called in test_video_module.py, after either course_nav.go_to_sequential or video.show_captions. That is where the synchronization should be performed.

In particular, these lines looks pretty suspect to me: 
* https://github.com/edx/edx-platform/blob/5e972b2a66e214c53372bdb4d41a71215af9562f/common/test/acceptance/pages/lms/course_nav.py#L128
* https://github.com/edx/edx-platform/blob/5e972b2a66e214c53372bdb4d41a71215af9562f/common/test/acceptance/pages/lms/course_nav.py#L213
* And maybe (I'm not sure if clicking the transcript button triggers an ajax call) https://github.com/edx/edx-platform/blob/8a2d989369c8887ce425e9b8485812cfd56dad3b/common/test/acceptance/pages/lms/video/video.py#L353